tests/python/gaia-ui-tests/gaiatest/form_controls/header.py

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from abc import ABCMeta, abstractproperty
+from marionette_driver import By, Wait, expected
+from gaiatest.form_controls.form_control import Widget
+
+
+class GaiaHeader(Widget):
+    # This represents the gaia-header element
```

Comment:
Well, it might seem obvious, but with BinaryControl, I just don't know anymore when I should use BinaryControl, GaiaBinaryControl, HtmlBinaryControl or InvisibleHtmlBinaryControl.